src/netex-convertor/sharedHelpers.ts

```diff
@@ -216,52 +212,46 @@ export const replaceIWBusCoNocCode = (nocCode: string): string => {
 
 export const getCoreData = (
     operators: Operator[],
-    matchingData: PointToPointTicket | PeriodTicket | SchemeOperatorTicket,
+    ticket: PointToPointTicket | PeriodTicket | SchemeOperatorTicket,
 ): CoreData => {
-    const pointToPoint: boolean = matchingData.type === 'single' || matchingData.type === 'return';
+    const pointToPoint: boolean = ticket.type === 'single' || ticket.type === 'return';
 
     if (pointToPoint) {
-        const pointToPointMatchingData: PointToPointTicket = matchingData as PointToPointTicket;
+        const pointToPointTicket: PointToPointTicket = ticket as PointToPointTicket;
```

Comment:
When the type guards are in place we shouldn't need this variable anymore and we could just use `ticket`